doc/ref_arch/README.md

```diff
@@ -35,3 +35,20 @@ We need to distinguish between architectural principles and architectural requir
 1. **Security compliance:** To ensure the architecture follows the industry best security practices and is at all levels compliant to relevant security regulations.
 
 1. **Resilience and Availability:** To allow High Availability and Resilience for hosted VNFs, and to avoid Single Point of Failure.
+
+<a name="ra-relationship-scope"></a>
+## Reference Architectures Relationships and Scope
+
+<p align="center"><img src="./figures/ref_arch_relationships.png" alt="Scope and Relationships" title="Scope and Relationships" width="80%"/></p>
+<p align="center"><b>Figure 1:</b> Relationships between and scope of the CNTT Reference Architectures</p>
+
+
+RA1 is focussing on an OpenStack Reference Architecture that will support VM-based VNFs only (i.e. no containerised workloads), whilst delivering the NFVI and VIM requirements as outlined in the Reference Model.
+
+RA2 is adding to this by providing a Kubernetes Reference Architecture that will enable the support for Cloud native Network Functions (CNFs) that comprise of either containers or VMs (both of which would be managed through Kubernetes). Note that RA2 is not looking to support the running of VNFs through Kubernetes (this would likely invoke discussions relating to Kubernetes as a VIM, which is not appropriate at this time).
+
+Key notes:
+- The primary difference between VNF and CNF in this context is the maturity of the software to be able to make use of immutable infrastructure, not whether it is a VM or not
+- RA2 scope will include mention of Kubernetes cloud providers that may integrate with other clouds (i.e. other than OpenStack as defined in RA1)
+- For the purposes of a CNTT Reference Implementation / Reference Certification the only fully conformant implementation of Kubernetes will be a Kubernetes that is conformant with RA2 that is itself deployed on an OpenStack deployment that is conformant with RA1
```

Comment:
Sooo an RA2 compliant bare metal Kubernetes deployment is not possible?

Reply:
It is with Ironic, yes. Alternatively we need a bare metal cloud reference architecture. 
Otherwise if an RA2 k8s is deployed on something else, how can we be sure the RM requirements are being met by the NFVI?

Reply:
Alternatively we discuss this particular point further at TSM. It was only mentioned briefly yesterday.

Reply:
Some comments (and be aware, that I can not open the figure, so maybe I'm missing some context here):

- RA1 is not only Ironic, but a lot more. So if we require RA2 to be deployed on top of RA1 to run bare metal Kubernetes we will have a lot of services needed for VM management unused. 
- I have some memories from Antwerp on agreeing that we do not care what is "under" the Kubernetes layer of RA2. The RM requirements should be independent from the undercloud of any NFVI

Reply:
Good point re. Ironic. On the second point I'll have a brief look through the notes. Otherwise it would be good to get comments from other TSM attendees here. I think that as long as we are able to show traceability to the RM requirements using any cloud then I agree with your view.

Reply:
From meeting 7/11/19: consensus reached on removing this last point and replacing with a comment stating that RA2 is standalone and does not *require* RA1 in order to be conformant.

Reply:
I have updated the content, including removal of these notes. In order for RA2 to be standalone, the points Gergely has made have been added to the new wording, I hope.